delegate_to_drep.sh

```
@@ -51,24 +52,24 @@ esac
 
 # Perform the transaction
 echo "Building transaction..."
-cardano-cli conway transaction build \
-  --tx-in $(cardano-cli query utxo --address $(< payment.addr) --testnet-magic 2 --out-file /dev/stdout | jq -r 'keys[0]') \
+$CARDANO_CLI conway transaction build \
+  --tx-in $($CARDANO_CLI query utxo --address $(< payment.addr) $NETWORK --out-file /dev/stdout | jq -r 'keys[0]') \
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)